scripts/release/README.md

```diff
@@ -30,6 +30,7 @@
 1. That's it! 
     - The content should now be released in GitHub.
     - The content should now be released in the MX Marketplace. Double check to verify.
+    - The minimumMXVersion should match the Mindex project version.
```

Comment:
Right now this is in the "that's it, these are the results" section. Please move this up, to the "things to do" sections.